docs/src/references/agent/operation-workflow.md

```diff
@@ -255,7 +255,35 @@ Data extracted from the command status topic and message payload can be passed a
 
 The script exit status and output is used to determine the next step for the command.
 - If the script cannot be launched or return a non-zero status, the command request is marked as __failed__.
-- If the script returns a json payload, this payload is injected into the previous message payload
-  (adding new fields, overriding overlapping ones, keeping previous unchanged ones).
-- If the script returns a json payload with a `status` field this status is used as the new status for the command.
+- If the script successfully returns, its standard output is used to update the command state payload.
+  - From this output, only the excerpt between a `:::begin-tedge:::` header and a `:::end-tedge:::` trailer is decoded.
+    This is done to ease script authoring. A script can emit arbitrary output on its stdout,
```

Comment:
Initially I was wondering why we couldn't just parse the last `{` and `}` chars instead of `:::begin-tedge:::` and `:::end-tedge:::`. But, I understand that this simplifies our parsing logic, if there are multiple JSON objects printed by earlier commands. But still, picking up the last JSON object should still be okay, as I can't imagine any script performing more steps after printing this status message, which is supposed to be the last thing that it does.

But, this simplification can be done later as well, as this contract is also acceptable.